### PR TITLE
Borg modules now require data research instead of anomaly research

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Borg modules? More like never used because not only does nobody do anomaly research but there isn't also any incentive to do so aside from "borgs are now more functional at their jobs"
:cl:
 * tweak: Silicon modules that used to require anomaly research now require level 5 data research.

### CODE REVIEW

**Comment by iFlashYou** [comment]
Tesla on anomaly 2 then we can talk

**Comment by boy2mantwicethefam** [comment]
cuz u a brotha imma bend t. knee to be fair though anomaly research still needs some love because the largest anomaly researches used to be the borg modules and the only anomaly research above 1 was a phazon glowstick, I have plans to make anomaly research useful tho I'm not sure how, but that's for another time

**Comment by Hoshi-chan** [comment]
You do know that the tesla energy relay is used for cheesing illegal research, too, not just the free mech energy, right?

This does make more sense though. Nobody's going to do anomaly research to help borgs, but they're definitely going to do it if it gets scientists their illegal shit.

**Comment by Accipitermaris** [comment]
Weren’t the new borg upgrades designed with the idea in mind that anomaly research was uncommon?

That was probably an err if so because it just resulted in this common sense change that also runs against that idea in the process.

**Comment by DamianX** [comment]
#17109 what changed?

**Comment by ShiftyRail** [comment]
@DamianX Probably the bargain about the Telsa thing.

**Comment by SonixApache** [comment]
borgcreep
the one thing keeping borgs from always having grippers is now cancelled

**Comment by Lunstra** [comment]
Definitely powercreep but I do think some of the modules need to be more available. The medical and service upgrades are only of any real use on low/deadpop where you likely to not have anyone at all working those departments, but as it stands now the upgrades are basically impossible to get on such pop, and in highpop you've likely got a medbay staff, kitchen and/or cargo to mass order pizza all day so their value drops off.

The engiborg grippers however, while their supposed used is to be able to set up all the engines, by the time they're available, even with this change, power is already up and effectively infinite so their only real purpose is to let a borg make canister bombs and fuck that noise, so they should probably stay behind the anomaly gate.

The only other thing I can add is while making grippers more available, especially for deadpop, is nice, maybe the requirement should be shifted to NT research? It's a little more involved that the roundstart R&D checklist, and top of the line upgrades for NT borgs being behind their experimental research makes a little more sense to me.

**Comment by Diavill** [comment]
unadultered borg creep PR which is somehow getting more upboats than the previous unadultered borg creep PR because of meme author

**Comment by aceedex** [comment]
Counterpoint: anomaly research is an extra hoop to jump through that isnt super interesting or interactive and things like the boeg upgrade wasnt locked bhind it before

**Comment by boy2mantwicethefam** [comment]
ya boi gonna answer concerns tho im sleeping so theyll be answered tomorrow, shoot, tho i dont expect people to change their minds about it

**Comment by Kurfursten** [comment]
Grippers were a mistake. But they were let to slide in on the basis "most of them will only extremely rarely appear so it's not going to impact things much". I'd rather see grippers taken out entirely. Stop trying to give them hands.

I don't really get how anomaly research isn't "interesting or interactive" when it's the only R&D research that requires you to do something resembling real science, do something dynamic where you find an artifact that could be anywhere out on the asteroid as opposed to braindead checklist following for 2 minutes at roundstart. It really feels like there's a confusion of tongues here because it's the rest of R&D that isn't interesting or interactive. It's not exactly Tetris.

**Comment by boy2mantwicethefam** [comment]
@Kurfursten i think the only gripper that isnt used for a very specific purpose is the engi gripper, actually allows it to make stuff that would need circuits the absolute mad lad, when was the last time you saw borgs with upgrade modules? anomaly research is done ridiculously far too rarely even tho according to the wiki you can get like 5-6 ano research from just printing and throwing papers into the da. theres been an rd overhaul once but the retards turned it into an actual idle game thats even more braindead than current rd, there is no salvation. at least this way theres an incentive for xenoarch to not erp with them weird items and instead bring em on station, no reason for xenarchs to even come back unless they wanna get the excav drill or the better diamond one, at least this way nerds can yell at them to come with their anom items to make the holy grail item that gives power without rechargers and starts illegal research

**Comment by jknpj** [comment]
@Kurfursten grippers were a NECESSARY mistake. They solved a long shot of bugs an issues with internal beakers and that's the sole reason i ported them. Everything else (gripper upgrades, organ gripper, magnetic gripper) is powercreep and i made it require effort exactly because of that. People complained about them being too hard to make so i made traders start with them, so traders have something rare to sell and the station has something that they want to buy. I'd rather rebalance some upgrades than this or a complete gripper removal simply because any would bring more issues than fix anything.

**Comment by Kurfursten** [comment]
I agree with both the sentiments that old borgo beakers/drinkingglasses were glitchy shit and also that anomaly research is super rare. I think that's a good thing, I don't want to see metal men taking over the kitchen every shift or making carbon engineers [even further irrelevant]. And I don't object to making tesla relays anomaly bound.

I fundamentally dislike the service grippers (kitchen, hydro) / organ gripper / mag gripper for a lot of reasons, not the least being that borgos are all access abominations full of creepy conveniences and immunities. If service ever gets lonely and wants extra labor they can grow dionas.

**Comment by SonixApache** [comment]
hi, mech user here, i object to making tesla relays anomaly bound since that means i will never see them ever again

File: code/modules/research/designs/mecha/tools.dm [hunk 1]
```diff
@@ -64,7 +64,7 @@
 	desc = "Tesla Energy Relay"
 	id = "mech_energy_relay"
 	build_type = MECHFAB
-	req_tech = list(Tc_MAGNETS = 4, Tc_POWERSTORAGE = 3)
+	req_tech = list(Tc_MAGNETS = 4, Tc_POWERSTORAGE = 3, Tc_ANOMALY = 2)
```
Comment (jknpj) [inline code/modules/research/designs/mecha/tools.dm]: Make it atomic

**Review by jknpj — DISMISSED** [review]

**Comment by ghost** [comment]
Isn't it crazy that all the thumbs up on this PR happened overnight after it was opened and no further thumbs up were submitted since then? haha

File: code/modules/research/designs/mecha/tools.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/modules/research/designs/mecha/tools.dm]: bruh why didnt you say it all this time

**Comment by boy2mantwicethefam** [comment]
@SonixApache forgot to use ur alt :^)

**Comment by GreenHairedAssistantRickysSmellyDick** [comment]
spires isnt pay to win retard

**Comment by jknpj** [comment]
This one, as it is right now, simply isn't happening.

I'll see what i can do to tweak upgrades(especially the medical one) to mitigate some issues that come with grippers upgrades gating things behind anomaly research.

**Comment by jknpj** [comment]
Feel free to open one about teslas though.

**Pull request closed without merging**